src/js/components/put-away/SplitLineModal.jsx

```diff
@@ -132,10 +151,18 @@ class SplitLineModal extends Component {
    * higher than available one.
    * @public
    */
-  isValid() {
+
+  isNegativeQuantity() {
+    return _.some(this.state.splitItems, items => _.toInteger(items.quantity) <= 0);
+  }
+
+  isQuantityHigherThanOriginalPutaway() {
     const qtySum = this.calculatePutAwayQty();
+    return qtySum > _.toInteger(this.props.putawayItem.quantity);
```

Comment:
just want to follow - do you have to use `_.toInteger`, because the quantity is in a string format?

Reply:
Yes, the quantity is in a string format

Reply:
hard to say in this specific place why it is a string, but I know that in some places we have quantity as string. My blind guess is that initially the quantity is type of number (fetched from DB), but then whenever you edit the value, it becomes the string, that's why I believe in many places we have some kind of validation, if the value given is not a `NaN`. I think that @alannadolny will explain this specific case, as it is just my blind guess. Am I right @alannadolny ?

Reply:
Yeah @kchelstowski you're right. The fetched quantity is type of number, after editing the value, it becomes the string:
![Screenshot from 2023-01-04 23-53-20](https://user-images.githubusercontent.com/83239466/210665694-d62f7560-4d3f-4d62-b0fc-fe34d3637927.png)
Someone set types of this quantity in propTypes and set it to string and number before
![Screenshot from 2023-01-05 00-00-04](https://user-images.githubusercontent.com/83239466/210666234-ffdc599e-44a7-4cab-98d3-228c4bb88a6a.png)